src/ApiManagement/ApiManagement.ServiceManagement/Commands/GetAzureApiManagementApi.cs

```diff
@@ -65,6 +66,13 @@ public class GetAzureApiManagementApi : AzureApiManagementCmdletBase
             HelpMessage = "If specified will try to get all Product APIs. This parameter is optional.")]
         public String ProductId { get; set; }
 
+        [Parameter(
+            ParameterSetName = FindByGatewayId,
+            ValueFromPipelineByPropertyName = true,
+            Mandatory = true,
+            HelpMessage = "If specified will try to get all Gateway APIs. This parameter is optional.")]
```

Comment:
please remove "This parameter is optional" it could be misleading.

Reply:
Done